src/components/Services.jsx

```
@@ -1,112 +1,104 @@
-      <h2 className="text-lg font-bold mb-3 text-white">{title}</h2>
-      <p className="text-sm font-poppins mb-4 text-white/80">{description}</p>
-      
-      <div className="space-y-2">
-        <p className="text-xs text-white/70 hover:text-white cursor-pointer transition-colors">
-          What's included in service?
-        </p>
-        <button className="w-full bg-white rounded-lg flex items-center justify-center py-2 hover:bg-gray-100 transition-all shadow-md hover:shadow-lg">
-          <span className="text-sm font-bold text-black">Schedule Discussion</span>
-        </button>
-      </div>
+const ServiceCard = ({ icon: Icon, title, description, classNames }) => (
```

ESLint

[error] 'icon' is missing in props validation

(react/prop-types)